install_config/install/advanced_install.adoc

```diff
@@ -402,6 +402,100 @@ meaning that it is available for placement of new pods. See
 xref:marking-masters-as-unschedulable-nodes[Configuring Schedulability on Masters].
 |===
 
+
+
+[[configuring-cluster-pre-install-checks]]
+=== Configuring Cluster Pre-install Checks
+
+Pre-install checks are a set of diagnostic tasks that run as part of the
+*openshift_health_checker* Ansible role. They run prior to an Ansible
+installation of {product-title}, ensure that required inventory values are set,
+and identify potential issues on a host that can prevent or interfere with a
+successful installation.
+
+Checks may have one or more tags, which allow for groups of related checks to be
+run together.
```

Comment:
This is unclear: there's no explanation of how/if these tags can be used in practice. I believe that this sentence can be removed.